apps/api/src/domain/models/bibliographic-reference/entities/JournalArticleBibliographicReferenceData.ts

```diff
@@ -0,0 +1,76 @@
+import {
+    ArrayNotEmpty,
+    IsOptional,
+    IsStringWithNonzeroLength,
+    IsUrl,
+} from '../../../../../../../libs/validation/src';
+import { DTO } from '../../../../types/DTO';
+import { isNullOrUndefined } from '../../../utilities/validation/is-null-or-undefined';
+import BaseDomainModel from '../../BaseDomainModel';
+import { IBibliographicReferenceData } from '../interfaces/IBibliographicReferenceData';
+import { BibliographicReferenceType } from '../types/BibliographicReferenceType';
+import Creator from '../types/Creator';
+
+export default class JournalArticleBibliographicReferenceData
+    extends BaseDomainModel
+    implements IBibliographicReferenceData
+{
+    readonly type = BibliographicReferenceType.journalArticle;
+
+    @IsStringWithNonzeroLength()
+    readonly title: string;
+
+    @ArrayNotEmpty()
+    readonly creators: Creator[];
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    readonly abstract?: string;
+
+    @IsStringWithNonzeroLength()
+    readonly date: string;
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    readonly publicationTitle?: string;
+
+    @IsOptional()
+    @IsUrl()
+    readonly url?: string;
+
+    @IsOptional()
+    @IsStringWithNonzeroLength()
+    readonly pages?: string;
```

Comment:
Use our internal `PageRange` type here, or something similar. I do not want unstructured data in our system. We can write the parsing logic in the import layer.

Reply:
We will deal with this later if we need it.